main.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+require_relative 'app'
+
+def main # rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength
+  app = App.new
+
+  status = true
+
+  while status
+    puts 'Welcome to School Library App!'
+    puts
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
```

Comment:
## OPTIONAL

- [ ] Please there is a duplication message, you can fix that. your App is working fine 👍 
-
![a](https://user-images.githubusercontent.com/63972670/174149202-293130b2-cd29-46ba-aabf-3d87fa672986.PNG)

Reply:
Dear @OLIPLICHE thank you for the feedback, I've just corrected it.